nautobot/extras/models/models.py

```diff
@@ -944,6 +953,187 @@ def log(
             logger.log(log_level, str(message))
 
 
+class ScheduledJobs(models.Model):
+    """Helper table for tracking updates to scheduled tasks.
+    This stores a single row with ident=1.  last_update is updated
+    via django signals whenever anything is changed in the ScheduledTask model.
+    Basically this acts like a DB data audit trigger.
+    Doing this so we also track deletions, and not just insert/update.
+    """
+
+    ident = models.SmallIntegerField(default=1, primary_key=True, unique=True)
+    last_update = models.DateTimeField(null=False)
+
+    objects = ExtendedManager()
+
+    @classmethod
+    def changed(cls, instance, **kwargs):
+        if not instance.no_changes:
+            cls.update_changed()
+
+    @classmethod
+    def update_changed(cls, **kwargs):
+        cls.objects.update_or_create(ident=1, defaults={"last_update": timezone.now()})
+
+    @classmethod
+    def last_change(cls):
+        try:
+            return cls.objects.get(ident=1).last_update
+        except cls.DoesNotExist:
+            pass
+
+
+class ScheduledJob(BaseModel):
+    """Model representing a periodic task."""
+
+    name = models.CharField(
+        max_length=200,
+        unique=True,
+        verbose_name="Name",
+        help_text="Short Description For This Task",
+    )
+    task = models.CharField(
+        max_length=200,
+        verbose_name="Task Name",
+        help_text='The Name of the Celery Task that Should be Run. (Example: "proj.tasks.import_contacts")',
```

Comment:
Use of Title Case for the `help_text` here and above seems a bit odd to me.

Reply:
Agreed.